dev-scheme/guile/guile-2.0.0.ebuild

```diff
@@ -52,7 +56,7 @@ src_configure() {
 }
 
 src_compile()  {
-	emake || die "make failed"
+	emake
```

Comment:
Good point. I originally kept it as is because of the emacs comment below, but I think eliminating it would be cleaner.